### PR TITLE
FullDataSet Update

### LINKED ISSUE #38: Update Data Model
- [x] update data model
- [x] update workflow to put datamodel into Redux

### PR DESCRIPTION
**Pull request by mmaelicke**
Closes #38

### CODE REVIEW

File: src/models/FullDataset.ts [hunk 1]
```diff
@@ -2,44 +2,65 @@
...
  * Do not use directly
  */
-export interface Indicator {
+export interface BaseIndicator {
+  short_name: string;
   value: number;
-  name: string;
-  description: string;
-  unit: "%" | "km / km^2" | "no / km^2";
+  description?: string;
```
Comment (mmaelicke) [inline src/models/FullDataset.ts:8]: I am not 100% happy here, as `Indicator` have a `description`, but `KeyFigures` have a `definition`. Is this intended? 
That means `KeyFigures` will not receive a description, right?

Comment (mmaelicke) [reply src/models/FullDataset.ts:8]: That means I would have to update the model to have the `description` only in `Indicator` and not in `KeyFigure`, where it is optional right now.

Comment (JesJehle) [reply src/models/FullDataset.ts:8]: Yes, kind of. I want to display a description of the indicator and what he is about. For the key figure, I would like to show the definition together with the data sources. The distinction of description and definition makes sense, here.

Comment (mmaelicke) [reply src/models/FullDataset.ts]: Alright, I updated the model accordingly

**Pull request merged**